resolwe_bio/processes/support_processors/qorts_qc.py

```diff
@@ -0,0 +1,105 @@
+"""QoRTs QC."""
+import os
+
+from resolwe.process import (
+    Cmd,
+    DataField,
+    FileField,
+    GroupField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class QortsQC(Process):
+    """QoRTs QC analysis."""
+
+    slug = 'qorts-qc'
+    name = "QoRTs QC"
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image': 'resolwebio/common:1.1.0',
+            },
+        },
+        'resources': {
+            'cores': 1,
+            'memory': 4096,
+        },
+    }
+    data_name = "QoRTs QC report ({{alignment|sample_name}})"
```

Comment:
Just curious: can we avoid jinja expressions in python processes?

Reply:
There is currently no other way of getting to `sample_name` (it works for `data_name` only). When this will change, we will be able to avoid jinja